code/__DEFINES/loadout.dm

```diff
@@ -99,6 +99,7 @@ GLOBAL_LIST_INIT(medic_gear_listed_products, list(
 		/obj/item/reagent_containers/hypospray/advanced = list(CAT_MEDSUP, "Hypospray", 2, "orange"),
 		/obj/item/reagent_containers/hypospray/advanced/big = list(CAT_MEDSUP, "Big hypospray", 10, "orange"),
 		/obj/item/clothing/glasses/hud/health = list(CAT_MEDSUP, "Medical HUD glasses", 2, "black"),
+		/obj/item/clothing/gloves/marine/corpsman = list(CAT_MEDSUP, "Advanced medical gloves", 4, "black"),
```

Comment:
```suggestion
		/obj/item/clothing/gloves/marine/corpsman = list(CAT_MEDSUP, "Advanced medical gloves", 2, "black"),
```
4 is quite the pretty penny for a very cool, yet not that useful item

Reply:
considering benos can now simply steal your defib, it's a pretty great space saving vs buying some chems which are a limited use thing.

Reply:
the only beno which can steal your defib is a primo rouny, and thats IF you had your defib out AND you were stunned WITH the defib out AND the rouny was around to grab your defib. Even with every chemical under the sun loaded into my belt as a corpsman, I still have space, PLUS a much more common occurrence then getting your defib snatched is getting your arm cut off by a hoonter, which keeps you from wearing gloves

Reply:
All xenos can drag items now.

Reply:
Really..? Fucks the point of a primo rouny then?

Reply:
rouny can steal the magharnessed gun out of your hand, then drag it to brazil.
For defibs? defibbing a dude, shrike flings you away then drags away defib. Can do the same with your medivac etc.

Reply:
in my like 150 hours of corpsman frontlining ive had my medivac stolen once but never my defib